src/main/resources/integration-jira-log-messages.properties

```diff
@@ -0,0 +1,4 @@
+integration.jira.url.base.invalid=Invalid base URL.
+integration.jira.url.base.invalid.solution=Inform a valid base URL.
+integration.jira.url.authorization.invalid=Invalid authorization callback URL.
```

Comment:
Partially done. I cannot say which config has to be changed since I don't know the source of that URL.